src/AmbientSounds/Models/AmbieJsonSerializerContext.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
+namespace AmbientSounds.Models
+{
+    /// <summary>
+    /// The <see cref="JsonSerializerContext"/> with type info for all JSON models used by Ambie.
+    /// </summary>
+    [JsonSerializable(typeof(FocusHistory))]
+    [JsonSerializable(typeof(FocusHistorySummary))]
+    [JsonSerializable(typeof(FocusTask[]), GenerationMode = JsonSourceGenerationMode.Metadata)] // Only used to deserialize
+    [JsonSerializable(typeof(IEnumerable<FocusTask>))]
+    [JsonSerializable(typeof(Video[]), GenerationMode = JsonSourceGenerationMode.Metadata)] // Only used to deserialize
+    [JsonSerializable(typeof(IList<Video>))]
+    [JsonSerializable(typeof(SyncData))]
+    [JsonSerializable(typeof(Sound[]), GenerationMode = JsonSourceGenerationMode.Metadata)] // Only used to deserialize
+    [JsonSerializable(typeof(List<Sound>))]
+    [JsonSerializable(typeof(RecentFocusSettings[]))]
+    public sealed partial class AmbieJsonSerializerContext : JsonSerializerContext
+    {
+    }
```

Comment:
Can you add 2 sets of comments above this:
1. One to describe why this is needed.
2. A second to describe how and when to add new stuff here.

I'm worried that when I need to make future changes, I wouldn't know how this is maintained, and I would just have to re-add the stuff you removed from the rd.xml file.

Reply:
Done 🙂